editor/components/block-toolbar/index.js

```diff
@@ -11,26 +11,28 @@ import BlockSwitcher from '../block-switcher';
 import BlockControls from '../block-controls';
 import BlockFormatControls from '../block-format-controls';
 
-function BlockToolbar( { block, mode } ) {
-	if ( ! block || ! block.isValid || mode !== 'visual' ) {
+function BlockToolbar( { block, mode, selectedBlockUIDs } ) {
+	if ( selectedBlockUIDs.length < 2 && ( ! block || ! block.isValid || 'visual' !== mode ) ) {
 		return null;
 	}
 
 	return (
 		<div className="editor-block-toolbar">
-			<BlockSwitcher uids={ [ block.uid ] } />
+			<BlockSwitcher uids={ selectedBlockUIDs.length > 1 ? selectedBlockUIDs : [ block.uid ] } />
```

Comment:
I wonder if we can't normalize this `selectedBlockUIDs` and `block` prop with a unique `selectedBlockUIDs` which would contain the selected blocks (multi-selection or not)

Reply:
That would make things a lot cleaner. Thanks!